tests/modules/localgov_forms_test/src/Driver/Database/FakeLts/Connection.php

```diff
@@ -29,4 +31,11 @@ public static function open(array &$connection_options = []) {
     return new StubPDO();
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function driverTransactionManager(): TransactionManagerInterface {
+    return new TransactionManager($this);
+  }
+
 }
```

Comment:
This seems to do the job:
```
diff --git a/tests/modules/localgov_forms_test/src/Driver/Database/FakeLts/Connection.php b/tests/modules/localgov_forms_test/src/Driver/Database/FakeLts/Connection.php
index 724d325..9a61fae 100644
--- a/tests/modules/localgov_forms_test/src/Driver/Database/FakeLts/Connection.php
+++ b/tests/modules/localgov_forms_test/src/Driver/Database/FakeLts/Connection.php
@@ -29,4 +29,11 @@ class Connection extends StubConnection {
     return new StubPDO();
   }
 
+  /**
+   * {@inheritdoc}
+   *
+   * Work-around for avoiding TransactionManager usage.
+   */
+  public function commitAll() {}
+
 }
```

Underlying [core issue](https://www.drupal.org/project/drupal/issues/2245905).

Reply:
Thanks it does seem to do the job.